### PR TITLE
Linux install

### PR DESCRIPTION
**Pull request by jburel**
Review dependencies installation on Debian 9

see https://trello.com/c/fF1IxQm5/17-package-installation-on-linux-install-walkthroughs

If we go ahead that will require some text adjustments
e.g. above the additional packages 
``# if install recommended/suggested is false``
Will that work for you @mtbc?
cc @mtbc @joshmoore

### CODE REVIEW

**Comment by jburel** [comment]
ubuntu14.04 does not have the package ``virtualenv`` and this is no longer a version that is exposed in our doc.
2 options:

 * remove ubuntu14.04 from the testing table
 * or split the shared file between ubuntu16.04 and ubuntu14.04

**Comment by jburel** [comment]
Going for the split

**Comment by mtbc** [comment]
I'm fine with anything that lets us test that our installation walkthrough still works without assuming installation of recommends, etc.

**Comment by jburel** [comment]
Doc PR opened openmicroscopy/ome-documentation#1845 to ease review

**Comment by rgozim** [comment]
@jburel Is it possible to build a version of the docs with your pull request changes included and if so can you show me how? I'd like to go over a compiled docs with your changes and try a step by step install on a clean Ubuntu 16.04 VM (that's where I encountered problems in the past).

Edit: I see you opened a pull request to ome-documentation. I'll just use that and see how it goes.

**Comment by jburel** [comment]
The doc PR will be included in Monday's build so it will be easier to follow
Thanks

File: linux/test/debian9_nginx/Dockerfile [hunk 1]
```diff
@@ -10,8 +10,8 @@ ARG JAVAVER=openjdk18
 ARG ICEVER=ice36
 ARG PGVER=pg96
 
-RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
- && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends
+RUN echo 'APT::Install-Recommends 0;' > /etc/apt/apt.conf.d/01norecommends \
+ && echo 'APT::Install-Suggests 0;' > /etc/apt/apt.conf.d/01norecommends
```
Comment (mtbc) [inline linux/test/debian9_nginx/Dockerfile]: I think this second one still needs to be an append. :smiley:

Comment (jburel) [reply linux/test/debian9_nginx/Dockerfile]: crap too much cleaning....

**Comment by mtbc** [comment]
Creation of `/etc/apt/apt.conf.d/01norecommends` looks fine now.

**Comment by jburel** [comment]
See commend from @rgozim https://github.com/openmicroscopy/ome-documentation/pull/1845#issuecomment-366717597

**Pull request merged**